src/coffee/services/base.coffee

```diff
@@ -40,6 +40,7 @@ class BaseService
     # Private: Container that holds request parameters such `id`, `query`, etc
     @_params =
       encoded: ['where', 'expand', 'sort']
+      plain: ['perPage', 'page']
```

Comment:
To know what parameters are also possible that are not encoded.
I use the array to loop trough when analyzing the queryString.